style.css

```diff
@@ -0,0 +1,315 @@
+    font-size: 20px;
+  }
+  
+
+
+.skills  {
+    color: white;
+    width: 380px;
+    max-width: 100%;
+    position: absolute;
+    top: 60%; /* Adjust the value to position the skills section below the about section */
+    left: 20%; 
+    transform: translateX(-50%);
+    text-align: center;
+
+  
+}
+
+.skills ul {
+   
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: row;
+    padding: 0;
+    list-style: none;
+    gap: 10px;
+    text-align: center; 
+    
+   
+}
+
+.skills li{
+    background-color: #05366f;
+    color: white;
+    border-radius: 5px;
+    padding: 5px 10px;
+    display: list-item;
+    text-align: -webkit-match-parent;
+    
+    
+}
+
+.contact-info {
+    width: 80%; /* Adjust the width as needed */
+    max-width: 800px; /* Adjust the maximum width as needed */
+    margin: 0 auto;
+    padding: 20px;  
+    border-radius: 10px;
+    text-align: center;
+    position: relative;
+    top: 410px; /* Adjust the top value as needed */
+    right: 600px; 
+
+   
+}
+
+.contact-info h2 {
+    color: white;
+    font-size: 24px;
+    margin-bottom: 5px;
+}
+
+.contact-info ul {
+   /* list-style: none;
+    padding: 0; */
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: row;
+    padding: 0;
+    list-style: none;
+    gap: 10px;
+    text-align: center;
+}
+
+.contact-info li {
+    color: white;
+    font-size: 18px;
+    padding: 5px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.contact-info li i {
+    margin-right: 5px;
+}
+
+#portfolio{
+    width: 80%; /* Adjust the width as needed */
+   max-width: 800px; /* Adjust the maximum width as needed */
+    margin: 0 auto;
+    padding: 20px;
+    border-radius: 10px;
+    color: white;
+    text-align: center;
+    position: relative;
+    top: 410px; /* Adjust the top value as needed */
+    right: 600px;
+
+    
+
+}
+
+.project {
+    color:white;
+    padding: 5px;
+    border-radius: 4px;
+    
+    transition: transform 0.3s ease-in-out;
+    
+  }
+  
+  .project:hover {
+    transform: translateY(-5px);
+  }
+  
+  .contact-info li:hover{
+   color: blue;
+  }
+               
+/* Add a media query for smaller screens */
+
+/* Add a media query for smaller screens */
+@media screen and (max-width: 768px) {
+    nav {
+        padding: 10px;
+    }
+
+    nav ul {
+        margin-top: 0;
+        justify-content: space-around;
+        flex-basis: auto;
+    }
+}
+
+/* Add a media query for smaller screens */
+@media screen and (max-width: 768px) {
+    .about {
+        max-width: 100%;
+    }
+}
+
+/* Add a media query for smaller screens */
+@media screen and (max-width: 768px) {
+    .skills,
+    .contact-info {
+        top: auto;
+        left: auto;
+        transform: none;
+    }
+}
+
+@media screen and (max-width: 768px) {
+    #portfolio {
+        top: auto;
+        left: auto;
+        transform: none;
+    }
+}

```

Comment:
You could use only one '@media screen and (max-width: 768px)' and then put styles under it since it is for the same size.